src/Sulu/Bundle/AdminBundle/Resources/js/components/AutoCompletePopover/AutoCompletePopover.js

```diff
@@ -44,9 +45,8 @@ export default class AutoCompletePopover extends React.Component<Props> {
     };
 
     handlePopoverClose = () => {
-        const {onSelect, suggestions} = this.props;
-        if (suggestions.length > 0) {
```

Comment:
closing the autocomplete popover by clicking somewhere else selected the first suggestion previously. with the changes in this PR, this would automatically select the first suggestion when a user clicks outside of the popover after he focused the input field once.